README.md

```diff
@@ -12,3 +12,19 @@ page](https://docs.github.com/en/get-started/writing-on-github/working-with-adva
 might help with the notation for mathematical expressions.
 
 $T(n) \in O(f(n)) \iff \exists c, n_0: T(n) \leq c \cdot f(n) \forall n \geq n_0$
+
+
+We assume $\log_{5}n$ = $\log_{a}n$  and   $\log_{2}n$  =  $\log_{b}n$
+
+$\log_{a}n$  $\ leq $  $\log_{a}b$  x $\log_{b}n$ 
+
+$\log_{a}n$ $\ leq $  $\frac{log_{x}b}{log_{x}a}$  x   $\frac{log_{x}n}{log_{x}b}$
+
+$\frac{log_{x}n}{log_{x}a}$ =  $\log_{a}n$ 
+
+Proof = $\log_{a}n$  $\ leq $   $\log_{a}n$
+
+Based on the above procedure, we can assume that $\log_{a}n$ is same as $\log_{b}n$
```

Comment:
But it's not! Logs with different bases clearly result in different numbers. $\log_2 10$ is different than $\log_3 10$.

Reply:
I've updated, using the correct logic. Please have a look at it and let me know. Thank you